torchquad/__init__.py

```diff
@@ -22,16 +25,15 @@
 from .utils.deployment_test import _deployment_test
 
 __all__ = [
+    "GridIntegrator",
```

Comment:
Do we want to expose these to the users? :thinking: 

Or is this with the idea of them also defining their own integrators based on them? Then it could be nice but we might want to either give an example or at least mention them in the docs :thinking: (if you prefer we can create a new issue for that)